spring/spring-cloud-config/src/test/java/io/quarkus/ts/spring/cloud/config/SpringCloudConfigIT.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.ts.spring.cloud.config;
+
+import static org.hamcrest.Matchers.is;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class SpringCloudConfigIT {
+
+    @Container(image = "${spring.cloud.server.image}", port = 8888, expectedLog = "Started ConfigServer")
+    static RestService spring = new RestService()
+            .withProperty("SPRING_PROPERTY_FILE", "resource::/config/application-SpringCloudConfigIT.properties")
```

Comment:
I was unable to find any mention of `SPRING_PROPERTY_FILE` anywhere. Can you perhaps point me in the right direction?

Reply:
This is used just to map the external file from the classpath to the container

Reply:
But how is it used by the container?
I mean, it obvoiusly works, I just don't see how.

Reply:
The property is ignored by the container, but the framework will detect that there is a resource that needs to be mapped into the container.

Reply:
I see. I didn't realize it's the external resources feature of the framework. Thanks for the explanation! :)